command-processor/tests/XmindHandlerTests.py

```diff
@@ -0,0 +1,13 @@
+from xmind.core.loader import WorkbookLoader
+from unittest import TestCase
+
+
+class LoaderTest(TestCase):
+    """Loader test"""
+
+    def test_init_get_abs_path_throws(self):
+        """test case when get_abs_path throws exception"""
+
+        with self.assertRaises(Exception) as ex:
+            WorkbookLoader('dd')  # create loader and waits for Exception
```

Comment:
Why are we testing WorkbookLoader here? I thought it was covered by tests in xmind project?

Reply:
It was done just to test that we can get xmind package directly from pip.
thank you for reviewing but PR is not ready for review yet :)